src/Aspire.Dashboard/Components/Controls/SummaryDetailsView.razor.cs

```diff
@@ -184,7 +184,7 @@ private void SetPanelSizes(float panel1Fraction)
 
     public async Task OnPageKeyDownAsync(KeyboardEventArgs args)
     {
-        if (_splitterRef is null || !args.ShiftKey)
+        if (_splitterRef is null || !args.ShiftKey || args.AltKey || args.CtrlKey)
```

Comment:
There is also the MetaKey to check for. TIL that's the Windows key, or whatever your device equivalent is.

https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/metaKey

Reply:
Put this logic into a helper method somewhere and reuse from all the OnPageKeyDownAsync places.

Reply:
Hrm. I didn't realize that was what MetaKey was... but also all the keyboard shortcuts I tried with the windows key seemed to get intercepted before the browser got them. But I'll add it anyway.

Reply:
Added that check.

> Put this logic into a helper method somewhere and reuse from all the OnPageKeyDownAsync places.

Done. Though have I mentioned I hate naming?